### ISSUE TITLE
Feature: Add faucet capabilities to devnet

### ISSUE THREAD

**Issue by Anmol1696** [cosmology-tech/starship#11]
## Overview
`values.yaml` has `faucet` key, but need to add the template `templates/faucet.yaml` using the faucet from confio.
```yaml
# cat values.yaml
faucet:
  enabled: false
  image: <docker image for facuet>
```

## Proposal
Faucet needs to be created for each of the chains defined in `chains` key in `values.yaml`. Either we create a deployment with a single pod for each chain, with minimal resources.

Mnemonics are stored in `charts/devnet/configs/keys.json`. Can use the first `jq -r "genesis[0].mnemonic" keys.json`.

We can use the `initContainers` for the setup.

**Comment by Anmol1696** [comment]
Bumping this to a requirement. Currently facing issues with transfering tokens from the mnemonic to address on test cases due to race condition caused by account missmatch error. This is forcing the tests to be run in serial, even though they can be run in parallel to some extent. Having a faucet service will help with that.

**Comment by Anmol1696** [comment]
Closed with #115
